ql/src/test/results/clientpositive/llap/cbo_rp_groupby3_noskew_multi_distinct.q.out

```diff
@@ -184,4 +184,4 @@ POSTHOOK: query: SELECT dest1_n123.* FROM dest1_n123
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@dest1_n123
 #### A masked pattern was here ####
-130091.0	260.182	256.10355987055016	98.0	0.0	142.9268095075238	143.06995106518906	20428.072876000002	20469.010897795593	79136.0	309.0
+130091.0	260.182	256.10355987055016	498.0	0.0	142.9268095075238	143.06995106518906	20428.072876000002	20469.010897795593	79136.0	309.0
```

Comment:
This diff is unexpected to me. Checking

Reply:
I found it is an issue of CBO return path + disabling map side aggregation.
https://github.com/apache/hive/pull/5245